primer3/wrappers.py

```diff
@@ -48,7 +56,7 @@
         os.environ['PRIMER3HOME'] = pjoin(LOCAL_DIR, 'src/libprimer3')
     except BaseException:
         raise OSError('PRIMER3HOME environmental variable is not set.')
-LIBPRIMER3_PATH = os.environ.get('PRIMER3HOME')
+LIBPRIMER3_PATH = os.environ['PRIMER3HOME']
```

Comment:
Seems like the right move given the perfunctory `pjoin` below; however, do we need to make any considerations now that a different exception would be raised in the case where `PRIMER3HOME` isn't set?

Reply:
I think this logic as modified works out as is.  `OSError` would get raised if there's an issue.  Lemme update this slightly

Reply:
Added an `if not os.path.isdir(LIBPRIMER3_PATH):` guard in the check.

Reply:
Great 👍